### DISCUSSION TITLE
autoplay?

### DISCUSSION THREAD

**Discussion by modeseven** [Tzahi12345/YoutubeDL-Material#418] [Q&A] [answered]
I love this project but I want to lay in bed.. and put on a subscription and have it keep playing the next video over and over? is that supported? I can't find it anywhere.

**Comment by Tzahi12345** [comment, marked as answer]
Not a bad idea, this isn't implemented but I'll convert this into an issue so we can keep track of the feature request.

See here: https://github.com/Tzahi12345/YoutubeDL-Material/issues/419

**Comment by Luckyyluck** [comment]
I have a similar ask to autoplay the self generated playlists.
I even dont find the subscriptions for which you obviousely already have implemented the "play all" button.
can you add something similar also to the custom playlists which I can generate from my downloaded files?
I dont see any benefit of such a playlists if it doesn't play :P

but overall a very great tool!
